tests/integration/test_package.py

```diff
@@ -45,7 +45,7 @@
         ],
     },
     'numpy': {
-        'version': '1.21.6',
+        'version': '1.23.3',
```

Comment:
had a similar issue to last time where I had to bump these slightly to get a version that included 3.11.